source/windows/windows_pki_utils.c

```diff
@@ -603,7 +603,7 @@ int aws_import_key_pair_to_cert_context(
             aws_raise_error(AWS_ERROR_SYS_CALL_FAILURE);
         }
 
-        if (i != cert_count - 1 || !add_result) {
+        if (i != 0 || !add_result) {
```

Comment:
Let's build a new unit test that stresses using a certificate-chain

Reply:
we could probably just use a cert chain for the testing-resources unit test certs.